packages/webpack/src/utils/config.ts

```diff
@@ -68,5 +68,5 @@ export function fileName (ctx: WebpackConfigContext, key: string) {
 
 export function getWebpackConfig (ctx: WebpackConfigContext): Configuration {
   // Clone to avoid leaking config between Client and Server
```

Comment:
```suggestion
  // Clone to avoid leaking config between Client and Server
  // TODO: rewrite webpack implementatation to avoid necessity for this
```